examples/csi_driver_install_vars.yaml

```diff
@@ -1,18 +1,22 @@
 ---
+ibmcloud_api_key: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"  #IBM Cloud API KEY
 service_instance_id: "bc895dc9-f626-41c0-bfd3-94a0d8942a5f"  #IBM Cloud PowerVS service instance id
 region: "mon"                                                #IBM Cloud PowerVS service instance region
 zone: "mon01"                                                #IBM Cloud PowerVS service instance zone
 
+resource_type: ""                                            #CSI driver resource type.
```

Comment:
Can this be "stable" in example. "" should not be allowed, is there a check?
Same for the next variable.